test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
+    ?a
+  end
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+l[x]
+l.call(x)
+
+  def b(x)
+    
+  end 
+
+
```

Comment:
__[development][rubocop]__
* Literal `%w(x b c)` used in void context.
* Extra blank line detected.

<!-- Comment by [SideCI](https://sideci.com) -->